### DISCUSSION TITLE
✅ Did paste stop working?

### DISCUSSION THREAD

**Discussion by dechimp** [isontheline/pro.webssh.net#666] [Q&A]
I used to long press the screen and paste would show up, however only "Export" shows up when I do that. I don't see any other way to paste. Did something change? Thanks

**Comment by isontheline** [comment]
@dechimp

No change at all.
The "Paste" menu is always here.

From which app are you copying text?

**Comment by dechimp** [comment]
It was a link from youtube.  Maybe some kind of mechanism YT is using to secure its content?

**Reply by isontheline** [reply]
I don't know but perhaps you're right.

**Comment by dechimp** [comment]
@isontheline You're right, it does seem to work from other apps.

**Reply by isontheline** [reply]
Thank you a lot for your feedback!
